test/parse_tests/test_parse_pt.py

```
@@ -0,0 +1,198 @@
+        # TODO - imperfect remainder
+        self.assertEqual(extract_duration("acorda-me daqui a três semanas, quatro dias e noventa segundos"),
+                         (timedelta(weeks=3, days=4, seconds=90),
+                          "acorda - me daqui a ,  e"))
+        self.assertEqual(extract_duration("10-segundos"),
+                         (timedelta(seconds=10.0), ""))
+        self.assertEqual(extract_duration("5-minutos"),
+                         (timedelta(minutes=5), ""))
+
+    def test_non_std_units(self):
+        self.assertEqual(extract_duration("1 mês"),
+                         (timedelta(days=DAYS_IN_1_MONTH), ""))
```

Ruff

Undefined name `DAYS_IN_1_MONTH`

(F821)